### DISCUSSION TITLE
Rendering with custom environment lighting

### DISCUSSION THREAD

**Discussion by luciancosw** [google-deepmind/mujoco#2108] [Asking for Help]
### Intro

Hi!

I am a researcher at discoverobotics, I use MuJoCo for my research on sim2real simulation.

### My question

I am wondering if Mujoco supports loading a custom environment lighting (in the formats of HDR map or cube map) for rendering, just like the features in Blender. After searching in the doc I have not found the solution yet.

Thanks in advance!

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
We are working on adding this, but it's early days, likely early next year.

**Reply by MarcelTorne** [reply]
Hi, I'm wondering if this is implemented already? Thanks!

**Reply by yuvaltassa** [reply]
We now load KTX files, is that what you need?

https://mujoco.readthedocs.io/en/latest/XMLreference.html#asset-texture-content-type

**Reply by yuvaltassa** [reply]
cc @haroonq
